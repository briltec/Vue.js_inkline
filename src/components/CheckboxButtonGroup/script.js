import IButtonGroup from '@inkline/inkline/components/ButtonGroup';
import ICheckableGroup from "@inkline/inkline/components/CheckableGroup";

export default {
    name: 'ICheckboxButtonGroup',
    extends: ICheckableGroup,
    components: {
        IButtonGroup
    }
};
