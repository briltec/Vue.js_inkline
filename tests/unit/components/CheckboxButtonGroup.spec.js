import { mount } from '@vue/test-utils';
import CheckboxButtonGroup from 'inkline/components/CheckboxButtonGroup';

describe('Components', () => {
    describe('CheckboxButtonGroup', () => {
        let wrapper;

        beforeEach(() => {
            wrapper = mount(CheckboxButtonGroup);
        });

        it('should be named correctly', () => {
            expect(CheckboxButtonGroup.name).toEqual('ICheckboxButtonGroup');
        });

        it('should render correctly', () => {
            expect(wrapper.html()).toMatchSnapshot();
        });
    });
});
