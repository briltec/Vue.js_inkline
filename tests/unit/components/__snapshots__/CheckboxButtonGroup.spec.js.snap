// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components CheckboxButtonGroup should render correctly 1`] = `<i-button-group-stub role="checkboxgroup" class="form-group -checkbox"></i-button-group-stub>`;
